homework/test/test-list.js

```diff
@@ -2,7 +2,14 @@ describe('Test > ', function() {
   beforeEach(function() {
   });
 
-  it('will test some pure functions', function() {
+  it('My pure function sin suc', function() {
     // Write any pure function assertion here.
+    assert.equal(Math.sin(Math.PI / 2), 1);
   });
+
+  it('My pure function sin failed', function () {
+    // Write any pure function assertion here.
+    assert.equal(Math.sin(Math.PI / 2), 0);
```

Comment:
Good. Although a better way is to define it in one manager and then call it here.